dev/MRTCore/mrt/mrm/include/mrm/common/MrmTraceLogging.h

```diff
@@ -25,6 +17,26 @@ void LogErrorInMemory(HRESULT hr, ULONG line, _In_ PCSTR filename, _In_ PCWSTR m
 #define LOG_ERROR_IN_MEMORY(hr, line, filename, message) __noop
 #endif
 
+class MrtRuntimeTraceLoggingProvider : public wil::TraceLoggingProvider
+{
+    IMPLEMENT_TRACELOGGING_CLASS(MrtRuntimeTraceLoggingProvider, "Microsoft.Windows.Mrt.Runtime",
+        // {706f5ec1-0c6b-54ee-4b6f-4cd329da9067}
+        (0x706f5ec1, 0x0c6b, 0x54ee, 0x4b, 0x6f, 0x4c, 0xd3, 0x29, 0xda, 0x90, 0x67));
```

Comment:
>(0x706f5ec1, 0x0c6b, 0x54ee, 0x4b, 0x6f, 0x4c, 0xd3, 0x29, 0xda, 0x90, 0x67)); [](http://example.com/codeflow?start=8&length=78)

is this just a placeholder?

Reply:
Nope, that's the GUID corresponding to the provider name.

Reply:
The name changed per this comment https://github.com/microsoft/WindowsAppSDK/pull/1171#discussion_r688835766. The GUID correspondingly changed.